packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -349,12 +354,12 @@ class FlutterPreSubmitFileComparator extends FlutterGoldenFileComparator {
 
   /// Decides based on the current environment if goldens tests should be
   /// executed as pre-submit tests with Skia Gold.
-  static bool isAvailableForEnvironment(Platform platform) {
+  static bool isRecommendedForEnvironment(Platform platform) {
```

Comment:
Curious about the name, why change it? `Recommended` seems a bit unclear to me. We must use the right comparator for the given environment, it being recommended makes it seem optional?

Reply:
Happy to go with another name. "isIntendedForEnvironment"? Maybe just "isForEnvironment"?

Reply:
I renamed it to `isForEnvironment` which seems to avoid the confusion of `Available` and `Recommended`, let me know what you think.